run_release.py

```diff
@@ -732,8 +732,10 @@ def run_add_to_python_dot_org(db: DbfilenameShelf) -> None:
 
     auth_info = db["auth_info"]
     assert auth_info is not None
+    # Do the interactive flow to get an identity for Sigstore
+    identity_token = subprocess.check_output(["python", "-m", "sigstore", "get-identity-token"]).strip().decode()
```

Comment:
Apparently the output of this line includes "Waiting for browser interaction...". We probably need to remove that